src/Models/ProductAttributeOption.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Rapidez\Statamic\Models;
+
+use StatamicRadPack\Runway\Traits\HasRunwayResource;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Cache;
+use Rapidez\Statamic\Traits\StoreIdTrait;
+
+class ProductAttributeOption extends Model
+{
+    use HasRunwayResource, StoreIdTrait;
+
+    protected $table = 'eav_attribute_option';
+
+    protected $primaryKey = 'option_id';
+
+    public $timestamps = false;
+
+    protected $appends = [
+        'admin_value',
+        'store_value',
+    ];
+
+    protected static function booting()
+    {
+        static::addGlobalScope(function (Builder $builder) {
+            $builder->join('eav_attribute', 'eav_attribute.attribute_id', '=', 'eav_attribute_option.attribute_id')
+                ->where('eav_attribute.entity_type_id', function ($query) {
+                    $query->select('entity_type_id')
+                        ->from('eav_entity_type')
+                        ->where('entity_type_code', 'catalog_product');
+                });
+
+            $builder->leftJoin('eav_attribute_option_value as admin_value', function ($join) {
+                $join->on('eav_attribute_option.option_id', '=', 'admin_value.option_id')
+                    ->where('admin_value.store_id', 0);
+            });
+
+            $builder->leftJoin('eav_attribute_option_value as store_value', function ($join) {
+                $join->on('eav_attribute_option.option_id', '=', 'store_value.option_id')
+                    ->where('store_value.store_id', static::getCurrentStoreId());
+            });
+
+            $builder->select([
+                'eav_attribute_option.option_id',
+                'eav_attribute_option.attribute_id',
+                'eav_attribute_option.sort_order',
+                'eav_attribute.attribute_code',
+                'eav_attribute.frontend_label as attribute_label',
+                'admin_value.value as admin_value',
+                'store_value.value as store_value',
+            ]);
+        });
+    }
+
+    public function attribute()
+    {
+        return $this->belongsTo(ProductAttribute::class, 'attribute_id', 'attribute_id');
+    }
+
+    public function AdminValue(): Attribute
```

Comment:
Do we want this to start with uppercase character? the docs say: 

```php
public function adminValue(): Attribute
```

would equal to 
```php
$model->admin_value
```